src/Controller/TranslateController.php

```diff
@@ -105,7 +105,8 @@ public function translate(
             'id' => 'upload-button-widget',
             'infusable' => true,
         ]);
-        if (!$session->get('logged_in_user')) {
+        $username = $session->get('logged_in_user');
```

Comment:
This session var is actually the whole array of userinfo, and not just the username.